Added 2 problems based on Boyer-Moore Majority Voting Algorithm(#78)

adityasays opened the pull request:
**added 2 problems** #78 
**algorithm to find elements that appear more than N/2 times in an array.**
**algorithm to find elements that appear more than N/3 times in an array.**
Can you please merge it under the hacktoberfest label

usmanbvp commented:
@adityasays  Kindly add the problems under array folder

usmanbvp commented:
@adityasays  don't create a seperate folder just add problems under the relevant folders

adityasays commented:
I have removed all the separate folders and added the problems inside the existing array folder , is there anything else to be corrected . Sorry for inconvenience .
